multi_imbalance/ensemble/mrbbagging.py

```diff
@@ -0,0 +1,108 @@
+from collections import Counter
+
+from scipy.stats import multinomial
+from sklearn.utils import resample
+import numpy as np
+
+
+class MRBBagging(object):
+    """
+    Multi-class Roughly Balanced Bagging (MRBBagging) is a generalization of MRBBagging for adapting to multiple
+    minority classes.
+
+    Reference:
+    M. Lango, J. Stefanowski:
+    Multi-class and feature selection extensions of RoughlyBalanced Bagging for imbalanced data.
+    J. J Intell Inf Syst (2018) 50: 97
+
+    Methods:
+    ----------
+    fit(x, y, k, learning_algorithm, undersampling=True)
+        Build a MRBBagging ensemble of estimators from the training data.
+
+    predict(data)
+        Predict classes for examples in data.
+    """
+    def __init__(self):
+        self.classifiers, self.classes, self.classifier_classes = dict(), dict(), dict()
+
+    def _group_data(self, x, y):
+        classes = set(y)
+        self.classes = {key: value for (key, value) in enumerate(classes)}
```

Comment:
`self.classes = dict(enumerate(classes))`
but is it really necessary to keep id:name map?

Reply:
It's for prediction matrix purposes.